app/search_result/search_result.component.ts

```diff
@@ -89,10 +90,19 @@ export class SearchResultExemptionComponent implements OnInit {
 export class SearchResultProcurementComponent implements OnInit {
   @Input() item: DocResultEntry;
   details: string;
+  entity_link: string;
+  valid_link: boolean;
```

Comment:
This is not related to subject of this PR, but - I think it will be better to follow a single code style. In this case - using `camelCase` for variables, properties and methods and `SCREAMING_SNAKE_CASE` - for constants (`snake_case` is rarely used in javascript world)